apps/docs/components/Playground.js

```diff
@@ -73,14 +77,36 @@ export default function Playground() {
     console.log('Waiting for server-ready event...');
     containerInstance.on('server-ready', (port, url) => {
       console.log('server-ready', port, url);
-      // TODO: Figure out hot reloading
-      // TODO: Figure out how to start server *after* build
-      setTimeout(() => {
-        setUrl(url);
-      }, 5000);
+      setUrl(url);
     });
   };
 
+  const updateFiles = async () => {
+    const containerInstance = instance.current;
+    const filePath = './src/app.jsx';
+    const updatedCode = code;
+    await containerInstance.fs.writeFile(filePath, updatedCode);
+    await wcSpawn(containerInstance, 'node', ['generateCSS.js']);
```

Comment:
I added a new `isUpdating` state and set it to true during this processing.

```suggestion
    const updatedCode = code;
    setIsUpdating(true);
    await containerInstance.fs.writeFile(filePath, updatedCode);
    await wcSpawn(containerInstance, 'node', ['generateCSS.js']);
    setIsUpdating(false);
```